test/sandbox.jl

```diff
@@ -17,6 +17,27 @@ test_test(fn; kwargs...)       = Pkg.test(;test_fn=fn, kwargs...)
             @test true
         end
     end end
+    temp_pkg_dir() do project_path; mktempdir() do tmp
+        copy_test_package(tmp, "BasicSandbox")
+        Pkg.activate(joinpath(tmp, "BasicSandbox"))
+        test_test(; julia_args=["--handle-signals=yes", "--warn-overwrite=no", "--math-mode=ieee"]) do
+            @test true
+        end
+    end end
+    temp_pkg_dir() do project_path; mktempdir() do tmp
+        copy_test_package(tmp, "RuntestsArgs")
+        Pkg.activate(joinpath(tmp, "RuntestsArgs"))
+        test_test(; test_args=["a", "b", "c"]) do
+            @test true
+        end
+    end end
+    temp_pkg_dir() do project_path; mktempdir() do tmp
+        copy_test_package(tmp, "RuntestsArgs")
+        Pkg.activate(joinpath(tmp, "RuntestsArgs"))
+        test_test(; test_args=["a", "b", "c"], julia_args=["--handle-signals=yes", "--warn-overwrite=no", "--math-mode=ieee"]) do
+            @test true
+        end
+    end end
```

Comment:
Tests here should test the actual API; e.g. with `Pkg.test()`

Reply:
Done.